ibis/tests/all/test_client.py

```diff
@@ -6,6 +7,32 @@
 from ibis.tests.backends import BigQuery, PySpark
 
 
+@pytest.mark.xfail_unsupported
+def test_load_data(backend, con):
+    table_name = 'test_salary'
+
+    sch = ibis.schema(
+        [
+            ('first_name', 'string'),
+            ('last_name', 'string'),
+            ('department_name', 'string'),
+            ('salary', 'float64'),
+        ]
+    )
+
+    df = pd.DataFrame(
+        {
+            'first_name': ['A', 'B', 'C'],
+            'last_name': ['D', 'E', 'F'],
+            'department_name': ['AA', 'BB', 'CC'],
+            'salary': [100, 200, 300],
+        }
+    )
+    con.drop_table(table_name, force=True)
+    con.create_table(table_name, schema=sch)
```

Comment:
Is there any way to create a table from the inferred pandas schema? That is, can one automatically come up with a schema given a dataframe, thereby enforcing that they are consistent?

Reply:
I will take a look about that.

Reply:
current `create_table` works with schema or an expression. so maybe using pandas backend we can pass the expr from that.

Reply:
for passing a dataframe to create_table, maybe would be good to add a `data` parameter. I will open an issue for that.